oddish.ts

```diff
@@ -424,7 +424,7 @@ const run = async () => {
 
     } else if (core.getInput("branch-to-next") === branch) {
       npmTag = "next";
-    } else if (customTag && customTag !== 'latest' && customTag !== 'next') {
+    } else if (customTag && customTag !== 'latest' && customTag !== 'next' && core.getInput("branch-to-custom-tag") === branch) {
```

Comment:
```suggestion
    } else if (customTag && core.getInput("branch-to-custom-tag") === branch) {
```

This propbably should be the first condition of the if-chain?